File: Results/DER/Translation/translation_output/misleading_test/deepseekcoder/codenet/java/python-sanitized/p02400.py
import math

def calculate_circle_properties(r):
    pi = math.pi
    x = r*r*pi
    y = 2*r*pi
    return round(x, 6), round(y, 6)

r = float(input())
x, y = calculate_circle_properties(r)
print(x, y)

File: Results/DER/Translation/translation_output/misleading_test/gpt-4/codenet/python/java-sanitized/p02400.java
import java.util.Scanner;

public class p02400 {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double r = scanner.nextDouble(); // Read the radius from the input

        double s = r * r * Math.PI; // Calculate the area of the circle
        double l = r * 2 * Math.PI; // Calculate the circumference of the circle

        System.out.printf("%.5f %.5f%n", s, l); // Print out the area and the circumference
    }
}